tools/update-eslint.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# Shell script to update ESLint in the source tree to the latest release.
+
+# Depends on npm and node being in $PATH.
+
+# This script must be be in the tools directory when it runs because it uses
+# $BASH_SOURCE[0] to determine directories to work in.
+
+cd "$( dirname "${BASH_SOURCE[0]}" )"
+rm -rf eslint
+
+# eslint-plugin-markdown is pinned at 1.0.0-beta.4 until there is a release
+# that fixes https://github.com/eslint/eslint-plugin-markdown/issues/69.
+npm install --no-bin-links --production eslint@latest eslint-plugin-markdown@1.0.0-beta.4
+rm package-lock.json 
+
+# Install dmn if it is not in path.
+type -P dmn || npm install -g dmn
+
+# Use dmn to remove some unneeded files.
+dmn -f clean
+
+# Move stuff to where it goes in our source tree.
+mv node_modules/eslint eslint
```

Comment:
I used `--global-style` to avoid having to move things in node_modules. The problem is I don't know how to install the markdown plugin this way.